server/controllers/userController.js

```diff
@@ -0,0 +1,88 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import Cryptr from 'cryptr';
+import model from '../models';
+
+dotenv.config();
+
+const { Users } = model;
+const secret = process.env.SECRET;
+const cryptr = new Cryptr(secret);
+
+const userController = {
```

Comment:
Instead of `userController`, why not use `authController`. `authController` will handle user `sign in` and `sign up` while `userController`  can handle all the user interaction after they have been authenticated by the system.